addons/hr/static/src/scss/hr.scss

```diff
@@ -14,18 +14,9 @@
         }
     }
     .o_employee_avatar {
-        position: relative;
         width: fit-content;
-        display: flex;
-        flex-direction: column;
-        align-items: flex-start;
         .o_employee_availability {
-            position: absolute;
-            top: -5px;
-            right: -5px;
-            padding-bottom: 1px;
-            border-radius: 50%;
-            background-color: $o-view-background-color;
+            margin-right: -5px;
```

Comment:
`me-n2`? to be tested

Reply:
Even though `me-n2` is only one pixel further, -5px provides a better optical alignment with the border of the avatar IMO